### DISCUSSION TITLE
adoptopenjdk and the install command

### DISCUSSION THREAD

**Discussion by srwarner** [Homebrew/discussions#214] [Casks]
I tried the install command at (https://formulae.brew.sh/cask/adoptopenjdk#default) which says:
   "brew install --cask adoptopenjdk"
yet, I obtained errors when I tried this (latest version of home brew). Instead I used:
   "brew cask install adoptopenjdk"
and that worked.  I dont know who to contact, figured here is as good a place as any.

Sam

**Comment by SMillerDev** [comment]
It works for me on the latest version. What does `brew config` say?

**Reply by srwarner** [reply]
Hi SMillerDev,
   getting back to you late, asI am only seeing that you asked question today.  I resolved the install issue I was having when I posted the initial message, and had put the initial post up as a way to help others in case they ran into the same issue. 

   I can't say I am using the same version of brew now, as I did a few updates since posting the message.  But... assuming nothing has changed ... 

        Homebrew 2.6.0
        Homebrew/homebrew-core (git revision 10f27; last commit 2020-12-03)
        Homebrew/homebrew-cask (git revision 5bb4a; last commit 2020-12-03)

   
Sam

**Reply by SMillerDev** [reply]
Those are very old versions, I'm glad your Java issue is resolved but you should really `brew update` and probably also `brew upgrade`.
